plugins/views.py

```diff
@@ -324,32 +325,28 @@ def list_dir(request):
     except UserNotFoundError:
         # This user has no access to the underlying system and therefore
         # must not be able to get a file listing
-        return HttpResponse(
-            json.dumps(
-                {
-                    "status": "You are not allowed to see the folder listing",
-                    "folders": [],
-                }
-            )
```

Comment:
Nothing to complain here, but I do think there's more of that stuff in solr (are we using this at all). and history.

Reply:
I just checked the ones in history/views.py (there are no in solr.py). They look kind of...special..
They use this `json.dumps` syntax but do not put json into it but simple strings...I think I will tackle them in a separate PR as I am a bit afraid of breaking something